Library/Formula/harfbuzz.rb

```diff
@@ -35,6 +44,8 @@ def install
       --with-coretext=yes
     ]
 
+    system "./autogen.sh" if build.head?
```

Comment:
Can you put this above `system "./configure", *args` line instead of here. Thanks.